src/lib.rs

```diff
@@ -163,26 +166,23 @@ impl<K, V> LruCache<K, V> where K: PartialOrd + Ord + Clone, V: Clone {
         })
     }
 
-    /// Check for existence of a key
-    // FIXME: Deprecated in favor of the `contains_key` function defined below.
```

Comment:
Yeah, I took the opportunity to clean it up here. Do you want me to raise a version change PR?